docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,2915 @@
+* Explain Log Rate Spikes: Allow to continue failed stream {kibana-pull}143301[#143301]
+* Entity filter for the Notifications page {kibana-pull}142778[#142778]
+* Show an info callout for new notifications {kibana-pull}142245[#142245]
+* Adding dashboard custom url to lens created jobs {kibana-pull}142139[#142139]
+* Adds ML open API output to appendix {kibana-pull}141556[#141556]
+
+Management::
+Adds missing geo aggs to autocomplete in Console {kibana-pull}141504[#141504]
+
+Maps::
+* Adds the ability to invert color ramp and size {kibana-pull}143307[#143307]
+* Adds layer groups {kibana-pull}142528[#142528]
+* Adds the ability to hide or show all layers {kibana-pull}141495[#141495]
+
+Observability::
+* Adds kibana.alert.time_range field to Alert-As-Data mappings and populate it {kibana-pull}141309[#141309]
+* Alert summary widget new design {kibana-pull}141236[#141236]
+* Adds histogram support for avg, max, min, sum and percentiles {kibana-pull}139770[#139770]
+
+Platform::
+* Allow date functions in formula {kibana-pull}143632[#143632]
+* Adds option to disable cursor sync on dashboards {kibana-pull}143355[#143355]
+* Adds maxIdleSockets and idleSocketTimeout to Elasticsearch config {kibana-pull}142019[#142019]
+
+Security::
+* Adds a read-only mode to the User management screen for users with `read_security` cluster privilege {kibana-pull}143438[#143438]
+* Adds a read-only mode to the API keys management screen for users with `read_security` cluster privilege {kibana-pull}144923[#144923]
+* Adds `user.id` field to Kibana audit log {kibana-pull}141092[#141092]
+
+Uptime::
+* Allow using AND for tags filtering {kibana-pull}145079[#145079]
+* Adds monitor detail flyout {kibana-pull}136156[#136156]
+
+[float]
+[[fixes-v8.6.0]]
+=== Bug fixes
+Alerting::
+* Fixes logger text and fix bulk error type {kibana-pull}144598[#144598]
+* Flaky bulkDisable tasks functional test {kibana-pull}144405[#144405]
+* Adding back unknown outcome filter {kibana-pull}143546[#143546]
+* Fixing flaky test in x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alerts_list·ts {kibana-pull}142855[#142855]
+* Rule run history displays success with a message when the rule status is warning {kibana-pull}142645[#142645]
+* Elasticsearch query rule type allows SIZE: 0, but flags as error on re-edit {kibana-pull}142225[#142225]
+* Rules and Connectors: Current page in breadcrumbs shows as link {kibana-pull}141838[#141838]
+* Move save button into connector config form {kibana-pull}141361[#141361]
+
+APM::
+* Show recommended minimum size when going below 5 minutes {kibana-pull}144170[#144170]
+* Fixes ml permissions by removing usage of `canAccessML` {kibana-pull}143631[#143631]
+* Fallback to terms agg search if terms enum doesn’t return results {kibana-pull}143619[#143619]
+* Fixes bug that causes alert expression to not close {kibana-pull}143531[#143531]
+* Fixes apm.transaction_duration alert to aggregrate over service environment {kibana-pull}143238[#143238]
```

Comment:
```suggestion

APM::
* Show a recommended minimum size when going below 5 minutes {kibana-pull}144170[#144170]
* Fixes ML permissions by removing usage of `canAccessML` {kibana-pull}143631[#143631]
* Fallback to terms aggregation search if terms enum doesn’t return results {kibana-pull}143619[#143619]
* Fixes bug that causes alert expression to not close {kibana-pull}143531[#143531]
* Fixes `apm.transaction_duration` alert to aggregrate over service environment {kibana-pull}143238[#143238]
```

Reply:
Can you add these two bug fixes as well, please?

- https://github.com/elastic/kibana/pull/143453
- https://github.com/elastic/kibana/pull/144472

Reply:
Thank you! Added.

```suggestion
APM::
* Show a recommended minimum size when going below 5 minutes {kibana-pull}144170[#144170]
* Fixes ML permissions by removing usage of `canAccessML` {kibana-pull}143631[#143631]
* Fallback to terms aggregation search if terms enum doesn’t return results {kibana-pull}143619[#143619]
* Fixes bug that causes alert expression to not close {kibana-pull}143531[#143531]
* Fixes `apm.transaction_duration` alert to aggregrate over service environment {kibana-pull}143238[#143238]
* Fixes broken latency and services layout {kibana-pull}143453[#143453]
* Fixes metadata API environment filter {kibana-pull}144472[#144472]
```